src/components/overviewCard.jsx

```diff
@@ -103,12 +102,24 @@ const OverviewCard = ({ daoOverview, members, daoVaults }) => {
             {`View ${getTerm(customTerms, 'bank')}`}
           </Button>
           <Button
+            mr={6}
             onClick={() => history.push(`/dao/${daochain}/${daoid}/proposals`)}
             value='proposals'
             title={getTitle(customTerms, 'Proposals')}
           >
             {`View ${getTerm(customTerms, 'proposals')}`}
           </Button>
+          {daoMetaData?.boosts?.SNAPSHOT?.active && (
```

Comment:
We now have a hook for this that check if a boost is active:

https://github.com/HausDAO/daohaus-app/blob/develop/src/hooks/useBoost.jsx

you can do something like:

```
const { isActive } = useBoost();

...

{isActive('SNAPSHOT') && (
```


example:
https://github.com/HausDAO/daohaus-app/blob/develop/src/components/proposalList.jsx